railties/lib/rails/generators/rails/scaffold_controller/USAGE

```diff
@@ -12,7 +12,7 @@ Description:
 Example:
     `rails generate scaffold_controller CreditCard`
 
-    Credit card controller with URLs like /credit_card/debit.
+    CreditCards controller with seven RESTful actions.
```

Comment:
I tried to be consistent with the `controller` generator: https://github.com/rails/rails/blob/e5632f37f797fd3cc6e2f793d7c73e97a55a8d73/railties/lib/rails/generators/rails/controller/USAGE#L14